### PR TITLE
gce: revoke kubelet binding permissions

### LINKED ISSUE #53151: remove kubelet binding RBAC role
It is not required now that we have the node authorizer.

### PR DESCRIPTION
**Pull request by mikedanese**
and move the binding addon to EnsureExists, so new clusters will pickup
the new binding and old clusters will keep the old binding. The binding
is no longer required now that we are migrating to node authorizer.

fixes https://github.com/kubernetes/kubernetes/issues/53151

### CODE REVIEW

**Comment by mikedanese** [comment]
Gah, master kubelet is not happy.

**Comment by liggitt** [comment]
right... master kubelet doesn't use bootstrapping

**Comment by mikedanese** [comment]
Ok now I'm only removing the binding if the master kubelet is configured not to register.

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -1739,7 +1739,11 @@ function start-kube-addons {
   local -r dst_dir="/etc/kubernetes/addons"
 
   # prep addition kube-up specific rbac objects
-  setup-addon-manifests "addons" "rbac"
+  setup-addon-manifests "addons" "rbac/kubelet-api-auth"
+  setup-addon-manifests "addons" "rbac/kubelet-cert-rotation"
+  if [[ "${REGISTER_MASTER_KUBELET:-false}" == "true" ]]; then
```
Comment (tallclair) [inline cluster/gce/gci/configure-helper.sh]: How will this play with upgrades? I.e. what if the cluster has an old node running?

Comment (mikedanese) [reply cluster/gce/gci/configure-helper.sh]: Now we preserve the binding on upgrade clusters now and create an empty binding on new clusters.

**Pull request merged**